src/Microsoft.AspNetCore.Antiforgery/AntiforgeryOptions.cs

```diff
@@ -54,10 +54,11 @@ public string CookieName
         public PathString? CookiePath { get; set; }
 
         /// <summary>
-        /// The domain set on the cookie. By default its <c>null</c> which results in the "domain" attribute not being
-        /// set.
+        /// Configures the <see cref="CookieOptions"/> of the antiforgery cookies. Without additional configuration, the 
+        /// default values antiforgery cookie options are true for <see cref="CookieOptions.HttpOnly"/>, null for
+        /// <see cref="CookieOptions.Domain"/> and <see cref="SameSiteMode.Strict"/> for <see cref="CookieOptions.SameSite"/>.
         /// </summary>
-        public string CookieDomain { get; set; }
+        public Action<CookieOptions> ConfigureCookieOptions { get; set; }
```

Comment:
Added him as a reviewer and will speak with him about the breaking change. Can you review the rest of the PR?